mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -45,6 +46,7 @@
 import org.xenei.junit.contract.ContractTest;
 import org.xenei.junit.contract.IProducer;
 
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Lists;
```

Comment:
Matthieu said that we used to use ImmutableList instead of Lists.
Updated and removed Lists in the file